index.js

```diff
@@ -86,18 +87,21 @@ function render (indexUrl, output) {
   }
 
   win.webContents.on('did-finish-load', function () {
-    win.webContents.printToPDF(opts, function (err, data) {
-      if (err) {
-        console.error(err)
-      }
 
-      fs.writeFile(path.resolve(output), data, function (err) {
+    setTimeout(function(){ 
```

Comment:
Please add a space to `function ()` so it pass [standard](https://github.com/feross/standard) code check.